components/sankeydiagram/customization/labels.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Labels
+page_title: Sankey Diagram Labels
+description: Sankey Diagram Labels
+slug: sankey-diagram-labels
+tags: telerik,blazor,sankey,diagram,chart,labels
+published: True
+position: 5
+---
+
+# Sankey Diagram Labels
+
+The Sankey Diagram Lables are the names of the [nodes]{%slug sankey-diagram-nodes%} that can be displayed on the nodes or next to them. This article explains how to customize the labels in the UI for Blazor Sankey Diagram. The listed settings will be applied to all labels in the Sankey Diagram.
```

Comment:
the names of the **nodes** that can be displayed on the **nodes** or next to them - this is hard to understand and at first glance looks like an endless recursion. Consider rewording, for example.

```suggestion
The Sankey Diagram Labels represent the [node]{%slug sankey-diagram-nodes%} names. The labels can show over the nodes or next to them. This article explains how to customize the labels in the UI for Blazor Sankey Diagram. Each setting applies to all labels in the Sankey Diagram.
```